config/locales/js-en.yml

```
@@ -343,6 +343,7 @@ en:
 
     label_activate: "Activate"
     label_assignee: "Assignee"
+    label_assignee_is: 'Assignee of the work package is'
```

That phrasing sounds weird, and it also strange that you have to concant the string later when it is clear that you will always pass a name
```suggestion
    label_assignee_alt_text: "This work package is assigned to %{name}"
```